js/commerce_pos.js

```diff
@@ -140,4 +140,89 @@
       .trigger('blur');
   }
 
+  Drupal.behaviors.commercePosSalePay = {
```

Comment:
This is making #28 larger than it should be and probably a little confusing: this change and a few others are actually from #23, which Shabana is working off of but isn't merged in yet (it's not quite ready; comments in that PR).